friends.gemspec

```diff
@@ -35,5 +35,6 @@ Gem::Specification.new do |spec|
 
   spec.add_development_dependency "minitest", "~> 5.5"
   spec.add_development_dependency "minitest-proveit", "~> 1.0"
+  spec.add_development_dependency "pry"
```

Comment:
Totally fine to add whatever you like for development, of course (I add plenty of my own too!), but please remove this to keep the dependency list as clean as possible 😄

Reply:
Whoops, my bad! I will remove this 👍